roles/keycloak/defaults/main.yml

```diff
@@ -12,6 +12,9 @@ keycloak_is_upgrade: false
 
 keycloak_service_user: "keycloak"
 
+keycloak_enable_technology_preview: false  # this is for enabling any new experimental features
+keycloak_proxy_subpath: ""  #leave this blank "" if you don't want any (normally, you don't)
```

Comment:
1. Rename `keycloak_proxy_subpath` to `keycloak_base_url_path` 

```
# The base URL path for frontend and backend requests. Note this cannot be empty.
keycloak_base_url_path: "auth"
```

2. Change default definition of keycloak_proxy_host
```
keycloak_proxy_host: "http://localhost:{{ keycloak_bind_port }}/{{ keycloak_base_url_path }}"
```